packages/js/src/api/inbox-service.ts

```diff
@@ -141,8 +141,15 @@ export class InboxService {
     });
   }
 
-  fetchPreferences(): Promise<PreferencesResponse[]> {
-    return this.#httpClient.get(`${INBOX_ROUTE}/preferences`);
+  fetchPreferences(tags?: string[]): Promise<PreferencesResponse[]> {
+    const queryParams = new URLSearchParams();
+    if (tags) {
+      tags.forEach((tag) => queryParams.append('tags[]', tag));
+    }
+
+    const query = queryParams.size ? `?${queryParams.toString()}` : '';
+
+    return this.#httpClient.get(`${INBOX_ROUTE}/preferences${query}`);
```

Comment:
Updated the InboxService to send the `tags` to the `GET /inbox/preferences` endpoint.